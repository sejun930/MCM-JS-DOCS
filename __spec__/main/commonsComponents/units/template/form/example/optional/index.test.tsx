import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";

import _ExampleOptionalFormPage from "src/main/commonsComponents/units/template/form/example/optional";

export default describe("Example Optional", () => {
  // 스냅샷
  test("Snapshot", () => {
    const { container } = render(
      <RecoilRoot>
        <_ExampleOptionalFormPage
          code="<span></span>"
          codeIdx={0}
          content="테스트 코드"
          isOpen={false}
          changeOpenList={() => {}}
        />
      </RecoilRoot>
    );
    expect(container).toMatchSnapshot();
  });
});
