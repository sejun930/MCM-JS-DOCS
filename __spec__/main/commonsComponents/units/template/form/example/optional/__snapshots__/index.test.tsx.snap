// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example Optional Snapshot 1`] = `
<div>
  <section
    class="css-cnt23e"
  >
    <button
      class="css-qf2s45"
    >
      <div
        class="example-code-icon-button css-16pyd80"
      >
        <span
          class="css-1iywk7f"
          role="example-left-tem"
        >
          &lt;
        </span>
        <span
          class="css-9alhxk"
          role="example-fin"
        >
          /
        </span>
        <span
          class="css-1t3qs2b"
          role="example-right-tem"
        >
          &gt;
        </span>
      </div>
    </button>
    <div
      class="css-nef91a"
    >
      <div
        class="copy-wrapper css-wbm39"
      >
        <div
          class="copy-wrappertext_ css-gjwso2"
        >
          <pre
            class="css-1ppfysk"
          >
            <code>
                
              <span
                class="purple"
              >
                return (
              </span>
              
    
              <span />
              
  
              <span
                class="purple"
              >
                )
              </span>
              <span
                class="lightGray"
              >
                ;
              </span>
            </code>
          </pre>
        </div>
        <button
          class="css-dqlxgg"
        >
          <p
            class="mcm-p-unit css-pzlb5d"
          >
            Copy
          </p>
        </button>
      </div>
    </div>
  </section>
</div>
`;
